.docker/privnet-entrypoint.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/sh
-if test -f /privnet-blocks.acc.gz; then
-	gunzip /privnet-blocks.acc.gz
-	/usr/bin/neo-go db restore -i /privnet-blocks.acc
+
+BIN=/usr/bin/neo-go
+
+if [ -z "$ACC"]; then
+  ACC=/6000-privnet-blocks.acc.gz
```

Comment:
nit: this line uses two spaces for indentation which differs from the rest of the script.

Reply:
replaced `tab` with `spaces`